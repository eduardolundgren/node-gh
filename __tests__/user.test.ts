/**
 * Â© 2013 Liferay, Inc. <https://liferay.com> and Node GH contributors
 * (see file: CONTRIBUTORS)
 * SPDX-License-Identifier: BSD-3-Clause
 */

import { runCmd } from './runCmd'

describe('E2E: User Module Test', () => {
    it('List Current User `gh user`', done => {
        expect(runCmd('gh user')).toMatchSnapshot()
        done()
    })

    it('Logout Current User `gh user --logout`', done => {
        expect(runCmd('gh user --logout')).toMatchSnapshot()
        done()
    })

    it('Show just the current username `gh user --whoami`', done => {
        expect(runCmd('gh user --whoami')).toMatchSnapshot()
        done()
    })
})
