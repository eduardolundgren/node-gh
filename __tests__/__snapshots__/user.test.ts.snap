// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`E2E: User Module Test List Current User \`gh user\` 1`] = `
"You're logged in as protoEvangelion
"
`;

exports[`E2E: User Module Test Logout Current User \`gh user --logout\` 1`] = `
"Logging out of user protoEvangelion
"
`;

exports[`E2E: User Module Test Show just the current username \`gh user --whoami\` 1`] = `
"protoEvangelion
"
`;
